static/src/javascripts/projects/common/modules/commercial/user-features.js

```diff
@@ -112,8 +122,7 @@ const refresh = (): Promise<void> => {
  * This data may be stale; we do not wait for userFeatures.refresh()
  * @returns {boolean}
  * @returns {boolean}
- */
-const isPayingMember = (): boolean =>
+ */ const isPayingMember = (): boolean =>
```

Comment:
Nit: this should probably be:

```
*/ 
const isPayingMember = (): boolean =>
```

Reply:
We can also probably kill the JSDoc-style `returns` comments now we have Flow

Reply:
Not sure how that slipped through. I ran it through make fix to let prettier fix the formatting before committing.

Reply:
`@returns` deleted.